src/components/Planning/PlanningTable/PlanningRow/PlanningHead.tsx

```diff
@@ -75,21 +74,34 @@ const PlanningHead: FC<IPlanningHeadProps> = ({
   /**
    * Adds the currently clicked items id to the search params, expand the row and navigate to the new URL
    */
-  const onExpand = useCallback(() => {
-    handleExpand();
+  const onExpand = useCallback(
+    (event: any) => {
```

Comment:
Avoid using any type if possible, otherwise use // eslint-disable-next-line @typescript-eslint/no-explicit-any

Reply:
Refactored to use ReactMouseEvent<SVGElement | HTMLElement>